api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,48 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceEventsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppEventsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  // num4xxResponses: This counter is akin to event errors/drops.
+  //
+  // 4xx response for a device event bundles does not make sense, unless
+  // there is a programming error in EVE code that send device event bundles to cloud.
+  // But, app instances can disappear from cloud and any futher logs sent by EVE to cloud
+  // will result in a 4xx response back.
+  // Any time an app log bundles get rejected with 4xx response from cloud, num4xxResponses
+  // should be incremented by the number of events present in that bundle.
+  //
+  // totalAppLogInput = numAppEventsSent + numAppEventErrors + num4xxResponses
```

Comment:
Need this algebra for the device logs above as well.
I'd move the algebra to a comment before the messge LogMetric

Reply:
Done